e2e/package.json

```diff
@@ -13,16 +13,16 @@
     "@types/mocha": "^5.2.6",
     "@types/node": "^11.13.4",
     "@types/selenium-webdriver": "^3.0.16",
-    "chai": "^4.2.0",
+    "axios": "^0.17.1",
+    "chai": "^4.1.2",
     "chromedriver": "^2.46.0",
     "mocha": "^6.1.4",
     "selenium-webdriver": "^3.6.0",
-    "ts-node": "^8.0.3",
-    "typed-rest-client": "^1.2.0",
-    "typescript": "^3.4.3"
+    "ts-node": "^8.1.0",
+    "typescript": "^3.4.5"
```

Comment:
This is same as ts-node@8.1.0 . CQ check will be passed with no issue. Just waiting now.

Reply:
Tracking https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19662

Reply:
The CQ was resolved by license_cerified. So I mark this conversation resolved.